dask/tests/test_base.py

```diff
@@ -27,20 +27,15 @@
     collections_to_dsk,
     get_collection_name,
     replace_name_in_key,
+    clone_key,
 )
 from dask.core import literal
 from dask.delayed import Delayed
-from dask.utils import tmpdir, tmpfile, ignoring
-from dask.utils_test import inc, dec
+from dask.utils import tmpdir, tmpfile
+from dask.utils_test import dec, inc, import_or_none
 from dask.diagnostics import Profiler
 
 
-def import_or_none(path):
-    with ignoring(BaseException):
-        return pytest.importorskip(path)
-    return None
```

Comment:
I am working on getting the min-deps tests passing again in https://github.com/dask/dask/pull/7247 and running into some issues with the changes in this PR around `import_or_none`. Can you explain what the reasoning was for the change?

Reply:
Ah, sorry I missed your PR @jsignell. FWIW I think there might be some duplicate work going on in https://github.com/dask/dask/pull/7243 and https://github.com/dask/dask/pull/7247

Reply:
@jsignell I moved import_or_none to test_utils because it is now used by two separate modules. Not sure I understood your question though?